package.json

```diff
@@ -1,15 +1,58 @@
 {
   "name": "filmfinder",
   "version": "1.0.0",
-  "description": "v 0.0.1",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "webpack --config=configs/webpack.production.config.js",
+    "start": "webpack-dev-server --open --config=configs/webpack.development.config.js",
+    "test": "jest",
+    "test-update": "jest --updateSnapshot",
+    "test-eslint": "eslint '**/*.{js,jsx}' --quiet"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/aleksandrbelik/filmFinder.git"
   },
-  "author": "",
-  "license": "ISC",
-  "homepage": "https://github.com/aleksandrbelik/filmFinder#readme"
+  "author": "Aleksandr Belik",
+  "contributors": [
+    "Danila Loginov"
+  ],
+  "license": "UNLICENSED",
+  "homepage": "https://github.com/aleksandrbelik/filmFinder#readme",
+  "devDependencies": {
+    "@babel/core": "^7.4.0",
+    "@babel/preset-env": "^7.3.4",
+    "@babel/preset-react": "^7.0.0",
+    "babel-jest": "^24.5.0",
+    "babel-loader": "^8.0.5",
+    "babel-plugin-module-resolver": "^3.2.0",
+    "babel-plugin-transform-export-extensions": "^6.22.0",
+    "css-loader": "^2.1.1",
+    "eslint": "^5.15.3",
+    "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-import-resolver-babel-module": "^5.0.1",
+    "eslint-import-resolver-webpack": "^0.11.0",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
+    "eslint-plugin-react": "^7.12.4",
+    "express": "^4.16.4",
+    "file-loader": "^3.0.1",
+    "jest": "^24.5.0",
+    "jest-dom": "^3.1.3",
+    "react-test-renderer": "^16.8.4",
+    "react-testing-library": "^6.0.0",
+    "style-loader": "^0.23.1",
+    "webpack-cli": "^3.2.3",
+    "webpack-dev-server": "^3.2.1"
+  },
+  "dependencies": {
```

Comment:
Webpack & webpack plugins are not runtime dependencies, they should be placed within the "devDependencies" object. Switch appropriate eslint rule off for webpack configurations.

Reply:
@loginov-rocks removed aslant rule for webpack.
As for "webpack" and "webpack-merge" eslint can't pass test with this rules in "devDepencies". It is generate the follow errors:
2:15  error  'webpack-merge' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies
  3:17  error  'webpack' should be listed in the project's dependencies, not devDependencies        import/no-extraneous-dependencies

Reply:
If you want to lint development-related stuff (you should, btw), you can add the following line at the beginning:
```
/* eslint-disable import/no-extraneous-dependencies */
```
OR you can create `configs/.eslintrc` with the following content:

```
{
  "rules": {
    "import/no-extraneous-dependencies": "off"
  }
}
```

In any case development dependencies should never appear in the runtime dependencies list.

Reply:
@loginov-rocks fixed package json and added appropriate rule to eslint config